sspl_test/run_qa_test.sh

```diff
@@ -121,31 +125,44 @@ restore_cfg_services()
     # Restoring MC port to value stored before tests
     if [ "$SSPL_STORE_TYPE" == "file" ]
     then
-        port=$(sed -n -e '/primary_controller_port/ s/.*\= *//p' /etc/sspl.conf)
+        port=$(sed -n -e '/primary_controller_port/ s/.*\: *//p' /etc/sspl.conf)
         if [ "$port" == "$MOCK_SERVER_PORT" ]
         then
-            sed -i 's/primary_controller_ip='"$MOCK_SERVER_IP"'/primary_controller_ip='"$primary_ip"'/g' /etc/sspl.conf
-            sed -i 's/primary_controller_port='"$MOCK_SERVER_PORT"'/primary_controller_port='"$primary_port"'/g' /etc/sspl.conf
+            sed -i 's/primary_controller_ip: '"$MOCK_SERVER_IP"'/primary_controller_ip: '"$primary_ip"'/g' /etc/sspl.conf
+            sed -i 's/primary_controller_port: '"$MOCK_SERVER_PORT"'/primary_controller_port: '"$primary_port"'/g' /etc/sspl.conf
         fi
         # Removing updated system information from sspl_tests.conf
         # This is required otherwise, everytime if we run sanity, key-value
         # pairs will be appended which will break the sanity.
         # Also, everytime, updated values from /etc/sspl.conf should be updated.
-        sed -i 's/node_id='"$node_id"'/node_id=001/g' /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
-        sed -i 's/rack_id='"$rack_id"'/rack_id=001/g' /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
-        sed -i 's/site_id='"$site_id"'/site_id=001/g' /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
-        sed -i 's/cluster_id='"$cluster_id"'/cluster_id=001/g' /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
+        sed -i "s/node_id: $node_id/node_id: SN01/g" /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
+        sed -i "s/rack_id: $rack_id/rack_id: RC01/g" /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
+        sed -i "s/site_id: $site_id/site_id: DC01/g" /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
+        sed -i "s/cluster_id: $cluster_id/cluster_id: CC01/g" /opt/seagate/$PRODUCT_FAMILY/sspl/sspl_test/conf/sspl_tests.conf
+    elif [ "$SSPL_STORE_TYPE" == "confstor" ]
+    then
+        port=$(conf $common_config get "storage>$encl_id>controller>primary>port")
+        port=$(echo $port | tr -d "["\" | tr -d "\"]")
+        if [ "$port" == "$MOCK_SERVER_PORT" ]
+        then
+            conf $common_config set "storage>$encl_id>controller>primary>port=$primary_port"
```

Comment:
Keep a TODO to solve following problem which would need to address during Cortx provisioner integration
I think we should do set on a copy of this global config, so later when provisioner give actual global config backend (file based), we wont be able to SET there.

Reply:
Added